Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_hole.h

```diff
@@ -81,6 +83,24 @@ namespace Polygon_mesh_processing {
       \cgalParamExtra{If no valid triangulation can be found in this search space, the algorithm
                       falls back to the non-Delaunay triangulations search space to find a solution.}
     \cgalParamNEnd
+
+    \cgalParamNBegin{use_2d_constrained_delaunay_triangulation}
+      \cgalParamDescription{If `true`, use the 2D constrained Delaunay triangulation facet search space.}
+      \cgalParamType{Boolean}
+      \cgalParamDefault{`true`}
+      \cgalParamExtra{If no valid 2D triangulation can be found, the algorithm
+                      falls back to the 3D Delaunay triangulation search space and/or the general search space to find a solution.
+                      This parameter is a good choice for near planar holes.}
+    \cgalParamNEnd
+
+    \cgalParamNBegin{max_squared_distance}
+      \cgalParamDescription{The maximum squared distance between the vertices of
+                            the hole boundary and the least squares plane fitted to this boundary.}
+      \cgalParamType{double}
+      \cgalParamDefault{squared one quater of the hole bounding box height}
```

Comment:
quater -> quarter

In German you can concatenate nouns to get a new one, but even in German "hole bounding box height"  would be too much :>    What about   "of the height of the bounding box of the hole".   Three times of, but I think clearer.

Reply:
Good point. I will change it.